pkg/controller/storagecluster/reconcile.go

```diff
@@ -1190,6 +1190,11 @@ func newCleanupJob(sc *ocsv1.StorageCluster) *batchv1.Job {
 		"app": "ceph-toolbox-job-${FAILED_OSD_ID}",
 	}
 
+	const purgeOSDScript = `set -x
+if [[ \"$osd_status\" == \"up\" ]]; then echo \"OSD ${FAILED_OSD_ID} is up and running. Please check if you entered correct ID of failed osd!\"; else echo \"OSD ${FAILED_OSD_ID} is down. Proceeding to mark out and purge\"
+ceph osd out osd.${FAILED_OSD_ID} 
+ceph osd purge osd.${FAILED_OSD_ID} --force --yes-i-really-mean-it; fi`
+
```

Comment:
```suggestion
	const purgeOSDScript = `
set -x
if [[ "$osd_status" == "up" ]]; then 
  echo "OSD ${FAILED_OSD_ID} is up and running."
  echo "Please check if you entered correct ID of failed osd!"
else
  echo "OSD ${FAILED_OSD_ID} is down. Proceeding to mark out and purge"
  ceph osd out osd.${FAILED_OSD_ID} 
  ceph osd purge osd.${FAILED_OSD_ID} --force --yes-i-really-mean-it
fi`

```

Reply:
@xenolinux please use this, it has proper indent.

Reply:
okay